app/forms/common/Address.scala

```diff
@@ -31,6 +31,8 @@ case class Address(
 ) {
   def isDefined(): Boolean =
     fullName.nonEmpty || addressLine.nonEmpty || townOrCity.nonEmpty || postCode.nonEmpty || country.nonEmpty
+
+  def extractValues: Seq[String] = Seq(fullName, addressLine, townOrCity, postCode, country)
```

Comment:
Is this an `unapply` in disguise?

Reply:
Some kind, I don't have better idea how to handle some things. In this case I need to have a sequence to pass to the view. Our components handle how to display `String` or `Seq[String]`, it makes everything in next row. Probably naming for the method here and in all different models are wrong. I'm open for any idea how to do it in different way

Reply:
Moved to the view and handled there